webapp/channels/build/emoji/additional_shortnames.json

```diff
@@ -2200,6 +2200,10 @@
     "om_symbol":[
         "om"
     ],
+    "pray": [
+        "thanks",
+        "thank you"
```

Comment:
@M-ZubairAhmed should it be `thank_you`
in the spinwick, we are only seeing `:thanks`

Reply:
@yasserfaraazkhan it should be "thank you" with a space. It was working for me on the spinwick

Reply:
@esethna can you check again even because emoji names don't have spaces

Reply:
![image](https://github.com/mattermost/mattermost/assets/14333569/397c5af8-a194-42eb-80c4-bdeb8e67e276)

@M-ZubairAhmed

Reply:
@yasserfaraazkhan i added both `thank you` and `thank_you`